NEWS.md

```diff
@@ -1,4 +1,35 @@
-## 5.0.0 - Unreleased
+## 6.0.0 - Unreleased
+
+## 5.0.0 - Released
+The primary focus of this release was to implement check-in flow and accommodate increased flexibility in inventory integration.
```

Comment:
It would be nice to mention the mod-organizations storage change as it is a notable switch

Reply:
Updated